src/Files.App/ViewModels/UserControls/Widgets/DrivesWidgetViewModel.cs

```diff
@@ -204,43 +212,29 @@ private void ExecuteOpenPropertiesCommand(WidgetDriveCardItem? item)
 			flyout!.Closed += flyoutClosed;
 		}
 
-		private void ExecuteDisconnectNetworkDriveCommand(WidgetDriveCardItem? item)
```

Comment:
Is this command not being used? If it's not, we should look at the commit history to see what happened.

Reply:
Yeah, thought the same. I'll look into.

Reply:
@0x5bfa